Sources/Search/APIs/Parsed.php

```diff
@@ -188,6 +188,13 @@ public function getSize(): int
 		$this->size = 0;
 
 		if (Db::$db->title === POSTGRE_TITLE) {
+
+			// Postgres will throw an error if the tables don't exist, so check first
+			$search_tables = Db::$db->list_tables(Db::$db->name, Db::$db->prefix . 'log_search%');
+			if (array_intersect([Db::$db->prefix . 'log_search_dictionary', Db::$db->prefix . 'log_search_parsed'], $search_tables) == []) {
+				return $this->size;
+			}
+
```

Comment:
```suggestion
			// Postgres will throw an error if the tables don't exist, so check first
			$search_tables = Db::$db->list_tables(Db::$db->name, Db::$db->prefix . 'log_search%');
			
			if (array_intersect([Db::$db->prefix . 'log_search_dictionary', Db::$db->prefix . 'log_search_parsed'], $search_tables) == []) {
				return $this->size;
			}

```

Keeps PHP-CS-Fixer happy.